examples/tar_ls/tar_ls.e

```diff
@@ -54,23 +58,26 @@ feature {NONE} -- Implementation
 			-- List the contents of `file'
 		local
 			block: MANAGED_POINTER
-			parser: USTAR_HEADER_PARSER
+			l_parser: USTAR_HEADER_PARSER
```

Comment:
we tends to use either short name (1 to 3 characters) to name a local or argument variable.
Or prefix with  "l_" for local, and "a_" for arguments.
This "prefix" conventions is not welcome by everyone, but it has the benefit to show quickly which token are locals/arguments and which are features (attribute, function ,..)

this is also to avoid potential name conflict in descendant, if one want to had a `parser' attribute for instance or related ...